test/index.test.js

```diff
@@ -48,17 +48,17 @@ describe('creating Sentry release', () => {
 })
 
 describe('uploading files to Sentry release', () => {
-  const release = 'test-release'
-
-  afterEach(cleanUpRelease(release))
-
-  it('uploads source and matching source map', () =>
+  it('uploads source and matching source map', () => {
+    const release = 'test-source-and-map'
```

Comment:
Each test needs to use a different release name, or else the fixtures get crossed up